API/NoDocs/Microsoft.Extensions.Compliance.Classification/DataClassificationAttribute.cs

```diff
@@ -0,0 +1,14 @@
+// Assembly 'Microsoft.Extensions.Compliance.Abstractions'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Compliance.Classification;
+
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Interface | AttributeTargets.Parameter | AttributeTargets.ReturnValue | AttributeTargets.GenericParameter, AllowMultiple = true)]
+public class DataClassificationAttribute : Attribute
```

Comment:
It's currently not marked abstract because that triggers this warning: https://rules.sonarsource.com/csharp/RSPEC-1694/

Do you buy this warning? I think abstract or not makes no difference, so I can go either way.

Reply:
> Do you buy this warning?

I don't :smile:

But as you say, since there's only a protected ctor, it doesn't really matter either way.

Reply:
OK, leaving as-is so I don't need to add another warning suppression :-)